fr/index.html

```diff
@@ -0,0 +1,391 @@
+            principes d'accessibilité améliore l'expérience utilisateur globale et rend un site web plus facile à
+            utiliser.
+          </p>
+        </section>
+
+        <section class="myth">
+          <h2 class="myth__title">Rendre les sites web accessibles n'apporte aucune plus-value</h2>
+
+          <p>
+            Pour n'en citer que quelques-uns : plus d'utilisateurs, plus de revenus, une meilleure réputation, moins de
+            risques de poursuites judiciaires, des équipes produit plus compétentes et bien d'autres encore. En fin de
+            compte, il n'est pas toujours nécessaire d'avoir une raison rentable pour faire une bonne chose.
+          </p>
+        </section>
+
+        <section class="myth">
+          <h2 class="myth__title">L'accessibilité peut être rectifiée par des outils de surcouche d’accessibilité</h2>
+
+          <p>
+            Absolument pas. Il existe <a target="_blank" href="https://overlayfactsheet.com/">une initiative</a> signée
+            par des centaines de professionnels qui prône la suppression des outils de surcouche d'accessibilité pour le
+            web.
+          </p>
+        </section>
+
+        <section class="myth">
+          <h2 class="myth__title">
+            L'utilisation d'outils automatisés est suffisante pour rendre un site web accessible
+          </h2>
+
+          <p>
+            Ce serait tellement plus simple, mais c’est impossible. En général, les tests d'accessibilité automatisés ne
+            peuvent détecter qu’<a
+              href="https://marcysutton.com/evinced-automated-accessibility-testing"
+              target="_blank"
+              >entre 30 et 50 % de tous les problèmes d'accessibilité</a
+            >. Il existe même
+            <a
+              href="https://www.matuzo.at/blog/building-the-most-inaccessible-site-possible-with-a-perfect-lighthouse-score/"
+              target="_blank"
+            >
+              un site web
+            </a>
+            qui démontre comment on peut construire un site “conforme” d’après les résultats de tests automatisés, alors
+            que le site est totalement inaccessible.
+          </p>
+        </section>
+
+        <section class="myth">
+          <h2 class="myth__title">L'accessibilité ne concerne que les utilisateurs aveugles</h2>
+          <p>
+            <strong>Certes, l’accessibilité agit sur l’expérience des</strong> utilisateurs aveugles mais aussi sur
+            celle des utilisateurs sourds, des utilisateurs ayant des handicaps moteurs et cognitifs... Elle agit aussi
+            sur celle des utilisateurs âgés; bref, de tous les utilisateurs. Elle est essentielle pour certains mais
+            utile pour tous.
+          </p>
+
+          <a href="https://www.w3.org/WAI/fundamentals/accessibility-intro/" target="_blank"
+            >Plus de détails <span class="visually-hidden"></span
```

Comment:
@MarionCouesnon could you please add a myth name to `visually-hidden`.

Reply:
Done with 77c2a05d2459067877887858494997c996167cdd